.github/actions/run-backend-tests/action.yml

```diff
@@ -32,10 +29,12 @@ inputs:
 runs:
     using: 'composite'
     steps:
+        # Pre-tests
+
         - name: Stop/Start stack with Docker Compose
           shell: bash
           run: |
-              export CLICKHOUSE_SERVER_IMAGE=${{ inputs.clickhouse-server-image }}
+              export CLICKHOUSE_SERVER_IMAGE=clickhouse/clickhouse-server:${{ inputs.clickhouse-version }}
```

Comment:
This is going to cause a nasty merge conflict with ongoing work to upgrade clickhouse to 22.8

Also we had the repo in there intentionally - they've switched out repositories out on us once :)

Reply:
Do you mean just this line or the overall change?

I'm totally fine with reverting to the existing `clickhouse-server-image`. I remember the `yandex/clickhouse-server` image, but I suspect we're not at risk of ClickHouse being spun out of ClickHouse Inc., so seems like the verbose form is no longer needed. This is just a cosmetic change though, only so that PR check names fit in without overflowing ("CH clickhouse/clickhouse-server:22.3" is just too long).

As for the wider change, it seems like the backend workflow has some incorrect assumptions baked in, so that needs to be fixed:
- persons-on-events is only relevant to EE tests (hasn't been true for at least the 5 months since #10206)
- FOSS tests don't need to be ran concurrently (in reality the FOSS part is equally big since #10206)
- persons-on-events are secondary to JOIN-based analysis (used to be the case, but we now need to switch this around)

Reply:
Just this line

On wider change - yup, there's problems there, it's looking like this will help clear things up